Packs/SecneurXAnalysis/TestPlaybooks/Detonate_File_-_SecneurX_Analysis_-_Test.yml

```
@@ -0,0 +1,203 @@
+id: 973f1923-2915-4130-8401-0a5911f59565
```

Please change the value to the value of `name`
**Both values should be the same**

```suggestion
id: Detonate File - SecneurX Analysis - Test
```